apps/root/src/pages/dca/positions/components/positions-list/current-positions/components/position-card-button/index.tsx

```diff
@@ -117,11 +118,19 @@ const PositionCardButton = ({
       position.chainId
     );
 
+  const reconnectingWalletDisplay = getDisplayWallet(wallet);
+
   return (
     <StyledCallToActionContainer>
       {!walletIsConnected && (
         <StyledCardFooterButton onClick={openConnectModal} fullWidth size="large">
-          <FormattedMessage description="reconnect wallet" defaultMessage="Reconnect wallet" />
+          <FormattedMessage
+            description="reconnect wallet"
+            defaultMessage="Reconnect wallet{wallet}"
```

Comment:
parenthesis and spaces should be here

Reply:
This is just in case so we dont show Reconnect wallet (), because wallet could be undefined